proplet/docs/scripts/test-addition.sh

```
@@ -0,0 +1,328 @@
+		return 1
+	fi
+
+	log_info "✓ CDH is enabled"
+
+	# Check TEE detection
+	log_info "Checking TEE detection..."
+	ssh -p 2222 propeller@localhost "sudo journalctl -u proplet | grep -i 'tee' | tail -5"
+
+	# Check Attestation Agent
+	log_info "Testing Attestation Agent..."
+	local aa_response=$(ssh -p 2222 propeller@localhost "curl -s http://localhost:50002/aa/evidence?runtime_data=test" || echo "ERROR")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)